mayhem/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Build Stage
```

Comment:
Can we add a package stage here?

Reply:
done! there are quite a few shared libraries :sweat_smile: